docs/wiki/introduction/using-osqueryi.md

```diff
@@ -104,3 +104,5 @@ $
 
 The shell does not keep much state or connect to the **osqueryd** daemon.
 If you would like to run queries and log changes to the output or log operating system events, consider deploying a query **schedule** using [osqueryd](using-osqueryd.md).
+
+**Note:** Event producers are not started by default. If you would like to enable event-based tables, you must use the flag `--disable_events=false`. osqueryi uses an in-memory DB by default. To connect to an existing events DB, use the flag `--database_path=/var/osquery/osquery.db` (note that `osqueryi` may not connect to the same DB that `osqueryd` is connected to, so `osqueryd` should be killed before using this flag).
```

Comment:
Phasing:
1. We often refer to the event produces as "publishers".
2. Let's use "database" instead of DB :)
3. The embedded not within parens is confusing. In the above text the name of the processes is in **bold**, not scriptified; killed is perhaps the wrong word, maybe stopped?

Perhaps for that last note we should refer folks to the debugging wiki page?